### DISCUSSION TITLE
The meaning of numTriNonzeroIntAmbiguity.h5

### DISCUSSION THREAD

**Discussion by LDcrystal** [insarlab/MintPy#1137] [Q&A]
Hello Yun Jun, I would like to ask about the numTriNonzeroIntAmbiguity.h5 generated in the quick_overview step. Does the red area in the left picture represent the area with large unwrapping error? Does the larger red area mean more unwrapping errors? Does the vertical axis in the picture on the right represent the number of pixels corresponding to the corresponding Tint? How to understand the actual meaning of the picture on the right?
![7ee2b424926f6e997b1b3bb2b102310d](https://github.com/insarlab/MintPy/assets/105006947/a3faff83-acaa-4e12-831c-d4f2b61ce739)

**Comment by welcome** [comment]
👋  Thanks for opening your first issue here! Please filled out the template with as much details as possible. We appreciate that you took the time to contribute!
Make sure you read our [contributing guidelines](https://github.com/insarlab/MintPy/blob/master/docs/CONTRIBUTING.md).

**Comment by yunjunz** [comment]
Yes, to all your questions. Unwrapping errors are relative to the reference point. The right subplot is the histogram of the left subplot. You could refer to section 3.2 in Yunjun et al. (2019) for some discussions on this figure. Based on this figure, it looks like the stack of interferograms has many unwrapping errors, and most of them are "incoherent" unwrapping errors, caused by decorrelation. Therefore, you may want to improve the coherence of the dataset first, by either focusing on a smaller area, or increasing the filtering strength and number of looks.
